docs/learning/backend/SchemasAndElementsInTypeScript.md

```diff
@@ -28,7 +28,7 @@ The app can ensure that the underlying schema is imported by registering an onOp
 
 where the schema is:
 
-``` xml
+```xml
```

Comment:
Why does this one have to be formatted this way, but the others on the page are okay?

Reply:
doesn't have to be, I think I manually switched that one around.